File: CursoLogica/Linguagens/Python/Atividades/soma_vetor.py
qnt = int(input("Quantos números você vai digitar? "))
lista = [0 for x in range(qnt)]

for i in range(qnt):
    lista[i] = float(input("Digite um número: "))
print()

soma = 0
print("VALORES = ", end="")
for i in range(qnt):
    print(f"{lista[i]:.1f} ", end="")
    soma += lista[i]
print()

print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {(soma / qnt):.2f}")

File: CursoLogica/Linguagens/C/Atividades/soma_vetor.c
#include <stdio.h>

int main() {

    int x, i;
    double soma, media, vet[9];

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &x);

    for (i = 0; i < x; i++) {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    printf("\nVALORES = ");
    for (i = 0; i < x; i++) {
        printf("%.1lf ", vet[i]);
    }

    soma = 0;
    for (i = 0; i < x; i++) {
        soma = soma + vet[i];
    }
    printf("\nSOMA = %.2lf\n", soma);

    media = (double) soma / x;
    printf("MEDIA = %.2lf", media);


    return 0;
}
